inf-clojure.el

```diff
@@ -176,13 +178,37 @@ often connecting to a remote REPL process."
 (defcustom inf-clojure-generic-cmd "lein repl"
   "The command used to start a Clojure REPL outside Lein/Boot projects.
 
-Alternative you can specify a TCP connection cons pair, instead
+Alternatively you can specify a TCP connection cons pair, instead
 of command, consisting of a host and port
 number (e.g. (\"localhost\" . 5555)).  That's useful if you're
 often connecting to a remote REPL process."
   :type '(choice (string)
                  (cons string integer)))
 
+(defvar-local inf-clojure-repl-flavor nil
```

Comment:
Btw, probably we should simply use `type` instead of `flavor`. Seems more obvious to me.